src/relayserver/HttpServer.ts

```diff
@@ -18,9 +18,9 @@ export class HttpServer {
     this.app.use(bodyParser.json())
     /* eslint-disable @typescript-eslint/no-misused-promises */
     this.app.post('/', this.rootHandler.bind(this))
-    this.app.post('/getaddr', this.pingHandler.bind(this))
```

Comment:
i removed this because i don't think it makes sense to have this as a `POST` call, given that we already have it as a `GET` on the next line

Reply:
My only concern about this is that maybe the client uses this endpoint, we should check that just in case, anyway doesn't make sense to me neither

Reply:
agree, I don't like it either, but maybe it's used by the client, or somethning else